### DISCUSSION TITLE
Redirect to subdomain not working on v3

### DISCUSSION THREAD

**Discussion by onemoreahmad** [livewire/livewire#5861] [7. Bugs (archived)]
**Livewire ^3.0@beta**

When I redirect users to the dashboard on `subdomain` after login, Livewire ignores subdomain and  do the redirection on the same domain and then disply a 404 error. I think this happens because the redirection happens on the frontend (Javascript) and conflict with Cross-Origin Resource Sharing (CORS) ?

```
    function login() {
 
        $this->withValidator(function ($validator) {
            $validator->after(function ($validator) {
                if (!Auth::attempt(['email' => $this->email, 'password' => $this->password], $this->remember)) {
                    $validator->errors()->add('email', trans('auth.failed'));
                }
            });
        });
 
        $this->validate();

        return redirect()->route('dashboard.home', ['username' => auth()->user()->username]);
        
    }

```
This should redirect to something like (`https://user1.myapp.test/dashboard`) 
But I got a 404 because Livewire tries to redirect to (`https://myapp.test/dashboard`) 
The redirection happens on javascript side, I can see it in the network tab. 

<img width="1111" alt="Screen Shot 2023-07-22 at 4 05 37 AM" src="https://github.com/livewire/livewire/assets/1885785/0240ed81-fe6a-4098-bb02-dd03101ea9ff">

I tried to set navigate: false, to make a redirection from the server side not from Javascript but with no luck. 

`return $this->redirect(route('dashboard.home', $user->tenant->handle), navigate: false);`

**Comment by killjin** [comment]
This is a bug https://github.com/livewire/livewire/discussions/5803.
Cross domain fetch request failed.
